cypress/e2e/tests/pages/global-settings/peformance.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { PerformancePagePo } from '~/cypress/e2e/po/pages/global-settings/performance.po';
+import { SettingsPagePo } from '~/cypress/e2e/po/pages/global-settings/settings.po';
+import HomePagePo from '~/cypress/e2e/po/pages/home.po';
+import BurgerMenuPo from '~/cypress/e2e/po/side-bars/burger-side-menu.po';
+import ProductNavPo from '~/cypress/e2e/po/side-bars/product-side-nav.po';
+
+describe('Performance', () => {
+  // If we need to speed tests up these should be combined into a single `it` (so only one page load and one refresh is used)
```

Comment:
We're at the point where this probably needs to happen.

For this PR all inactivity tests could be folded in to one